.github/workflows/test.yml

```diff
@@ -56,15 +71,19 @@ jobs:
           path: ${{ github.workspace }}/openassetio
 
       - name: Install ctest dependencies
+        # Use wheels from feature branch for intermediate testing
+        # Even though we're only testing cpp here, the python imports
+        # still need to be valid to not error pytest
         run: >
           python -m pip install
-          -r tests/requirements.txt
           -r tests/ctest-requirements.txt
+          -r tests/requirements.txt
```

Comment:
[question] Is this comment still correct? As its not `python -m pip install ./deps/openassetio-*cp37*-manylinux_*_x86_64.whl`-ing? Is it intending to say that we're using the `Build OpenAssetIO`  generated package under `${{ github.workspace }}/openassetio/lib/python3.9/site-packages`? If so, L86 might be a good place for the comment?

Reply:
I had removed the damn thing ... but in my attempt to clean up the 30+ fixup CI mess for review, ended up reintroducing it.

You're right though, a better comment can exist down below.
https://github.com/OpenAssetIO/OpenAssetIO-TraitGen/pull/51/commits/fdd3e6853490f4c25f94175e0d536bd003b31bf5